# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/qt6multimedia/hotfix-non-x86.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- qtmultimedia-everywhere-src-6.10.2/src/multimedia/CMakeLists.txt.vanilla	2026-02-20 18:13:00.368113354 +0100
+++ qtmultimedia-everywhere-src-6.10.2/src/multimedia/CMakeLists.txt	2026-02-20 18:16:42.762697314 +0100
@@ -157,6 +157,7 @@
     CONDITION (LINUX OR ANDROID) AND QT_FEATURE_library
     SOURCES qsymbolsresolveutils.cpp qsymbolsresolveutils_p.h)
 
+if (${CMAKE_SYSTEM_PROCESSOR} MATCHES "86")
 qt_internal_add_simd_part(Multimedia SIMD sse2
     SOURCES
         video/qvideoframeconversionhelper_sse2.cpp
@@ -177,6 +179,7 @@
     EXCLUDE_OSX_ARCHITECTURES
         arm64
 )
+endif()
 
 qt_internal_add_docs(Multimedia
     doc/qtmultimedia.qdocconf
 )
